src/pages/AzurePreview.js

```diff
@@ -129,7 +129,7 @@ const AzurePreviewUserForm = ({ value: formValue, onChange, onSubmit }) => {
 
     fieldset({ style: { padding: 0, border: 'none', margin: '2rem 0 0' } }, [
       h(FormLegend, { style: { marginBottom: '1rem' } }, [
-        'What do you want to do in Terra?',
+        'What do you want to do in Terra? *',
```

Comment:
Wasn't sure what was adding the `*` for the other required fields, but this looks okay to me

Reply:
👍 The FormLabel component is rendering the * for the other fields.